File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n=nums.length;
        for(int i=0;i<n-1;i++)
        {
            for(int k=i+1;k<n;k++)
            {
                if(nums[i]+nums[k]==target)
                {
                    return new int[]{i,k};
                }
            }
        }
        return new int[]{};
    }
}

File: 1-two-sum/two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        n=len(nums)
        mpp={}
        for i in range(n):
            number=nums[i]
            x=target-number
            if x not in mpp:
                mpp[number]=i
            else:
                return [mpp[x],i]
        return [-1,-1]
        
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        